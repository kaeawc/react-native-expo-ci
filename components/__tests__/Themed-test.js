import * as React from 'react';
import renderer from 'react-test-renderer';

import { Text, View, Image } from '../Themed';

it(`renders text correctly`, () => {
  const tree = renderer.create(<Text>Snapshot test!</Text>).toJSON();

  expect(tree).toMatchSnapshot();
});



it(`renders view correctly`, () => {
  const tree = renderer.create(<View>Snapshot test!</View>).toJSON();

  expect(tree).toMatchSnapshot();
});



it(`renders image correctly`, () => {
  const tree = renderer.create(<Image src="https://randomuser.me/api/portraits/lego/0.jpg" />).toJSON();

  expect(tree).toMatchSnapshot();
});
