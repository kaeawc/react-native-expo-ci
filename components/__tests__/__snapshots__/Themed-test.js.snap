// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders image correctly 1`] = `
<Image
  src="https://randomuser.me/api/portraits/lego/0.jpg"
  style={
    [
      undefined,
    ]
  }
/>
`;

exports[`renders text correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#000",
      },
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`renders view correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      undefined,
    ]
  }
>
  Snapshot test!
</View>
`;
